apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/Slate.kt

```diff
@@ -66,6 +66,14 @@ object SlateHelper {
             SakType.OMSTILLINGSSTOENAD -> {
                 when (behandling.vedtak.type) {
                     VedtakType.INNVILGELSE -> getJsonFile("/maler/vedlegg/oms_foerstegangsvedtak_innvilgelse.json")
+                    VedtakType.ENDRING -> {
+                        when (behandling.revurderingsaarsak) {
+                            RevurderingAarsak.INNTEKTSENDRING -> getJsonFile(
+                                "/maler/vedlegg/oms_revurdering_endring.json"
```

Comment:
Hvorfor skal vi ha filer for dette og ikke objekter?

Reply:
Kunne brukt objekter og kun lagret payloaden som fil da den fort kan bli veldig stor?

`listOf(BrevInnholdVedlegg(tittel = "Vedlegstittel", key=VEDLEGG_KEY, payload=getJSONFile("path/here"))`

Reply:
Føler det mer typesikkert men det er kanskje ikke mulig å få brukket det her?

Reply:
Enig det du sier med typesikkerhet - en refaktorering så må alle brevene manuelt endres. Men det gjelder vel i så fall alle brevene vi har så langt?